TicTacToe/UserInputConsole.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace TicTacToe
+{
+    public class UserInputConsole
+    {
+        private Move _moveInput = new Move(-1,-1);
+        private bool _hasGivenUp;
```

Comment:
I think it is important to be consistent conceptually, so a better name would be _playerHasQuit.